### PR TITLE
Loops (Actions) Destination

### PR DESCRIPTION
**Pull request by forstisabella**
### Proposed changes
Moved docs from original PR to editable PR.

### Merge timing
After approval by tcgilbert, docs

### Related issues (optional)
#5451

### CODE REVIEW

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+# The end name should be similar to `Slack  Destination`
+title: Loops
+hide-boilerplate: true
+hide-dossier: true
+---
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-loops/index.md:6]: I updated the title of the page to reflect that this destination is an Actions destination, added private/beta tags to reflect the status of this destination, and included the `id` from the Public API. 

```suggestion
---
title: Loops (Actions) Destination
hide-boilerplate: true
hide-dossier: true
beta: true
private: true
id: 63360a5fe290ca3fdfad4a68
---
```

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+{% include content/plan-grid.md name="actions" %}
+
+[Loops](https://loops.so?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target:="_blank"} is a modern email platform for SaaS, a better way to send marketing and transactional email.
+
+You can use this Segment integration to create and update your Loops contacts as well as trigger email sending via events.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-loops/index.md:12]:
```suggestion
You can use this Segment destination to create and update your Loops contacts and trigger email sending via events.
```

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+This destination is maintained by Loops. For any issues with the destination, [contact their Support team](mailto:help@loops.so).
+
+## Getting started
+
+1. From the Segment web app, click **Catalog**, then click **Destinations**.
+2. Search for “Loops (Actions)” in the Destinations Catalog, and select the destination.
+3. Click **Configure Loops**.
+4. Select an existing Source to connect to Loops.
+5. Generate an API key in Loops from [Settings > API](https://app.loops.so/settings?page=api){:target="_blank"}. Click "Generate key" then click the "Copy to clipboard" icon.
+6. Back in Segment, go to the Settings page inside your Loops destination and paste your API key then click "Save Changes".
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-loops/index.md:23]:
```suggestion
1. From the Segment web app, click **Catalog**, then click **Destinations**.
2. Search for “Loops (Actions)” in the Destinations Catalog, and select the destination.
3. Click **Configure Loops**.
4. Select an existing Source to connect to Loops and click **Next**.
5. On the Setup page, enter a name for your destination and click **Create destination**. 
6. Open Loops and generate an API key from [Settings > API](https://app.loops.so/settings?page=api){:target="_blank"}. Click "Generate key" then click the "Copy to clipboard" icon.
7. Open the Segment app, go to the Settings page inside your Loops destination, and paste your API key. Then, click "Save Changes".
```

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+## Create or update contacts
+
+You can create and update Loops contacts by using Segment's [Identify method](https://segment.com/docs/connections/spec/identify/){:target="_blank"}, like this:
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-loops/index.md:27]: You can use relative links to refer to pages inside the segment.com/docs/ subdomain!

```suggestion
You can create and update Loops contacts by using Segment's [Identify method](/docs/connections/spec/identify/), like this:
```

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+```javascript
+analytics.identify("test-user-a5h7xb", {
+  email: "adam@loops.so",
+  firstName: "Adam",
+  favoriteColor: "blue",
+  favoriteNumber: 42
+});
+```
+
+If the email address or user ID do not exist in your contacts, a new contact will be created. If the email address or user ID already exist, the existing contact will be updated with the data sent in the `identify()` call.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-loops/index.md:38]: Edited Identify call to match our [style guide](https://github.com/segmentio/segment-docs/blob/develop/styleguide.md#styling-the-segment-methods)

```suggestion
If the email address or user ID do not exist in your contacts, a new contact will be created. If the email address or user ID already exists, the existing contact will be updated with the data sent in the Identify call.
```

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+Go to the Mappings tab inside your Loops destination and click "New Mapping". Select "Create or update a contact".
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-loops/index.md:40]:
```suggestion
Go to the Mappings tab inside your Loops destination and click **New Mapping**. Select **Create or update a contact**.
```

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+It is important that you set up the data mapping properly in step 3 of the form. This is where you can select which data is sent on to Loops and into which fields. We provide an example default mapping in the form but you should make sure that you set up the mapping to capture the correct data into the correct fields (you may have some [custom fields in Loops](https://loops.so/docs/add-users/properties){:target="_blank"} that the default mapping doesn't cover, for example).
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-loops/index.md:42]: Hi @danrowden, @askkaz - is this form called the "Create or update a contact" form?
```suggestion
It is important that you set up the data mapping properly in step 3 of the "Create or update a contact" form. This is where you can select which data is sent on to Loops and into which fields. Loops provides an example default mapping in the form, but you should make sure that you set up the mapping to capture the correct data in the correct fields (you may have some [custom fields in Loops](https://loops.so/docs/add-users/properties){:target="_blank"} that the default mapping doesn't cover, for example).
```

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+Any custom fields you are using in Loops can be passed inside "Custom Contact Attributes".
+
+Once you have completed the mapping you can send a test event. Once submitted you can verify everything is set up correctly by looking for the contact on the [Audience](https://app.loops.so/audience){:target="_blank"} page in your Loops account.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-loops/index.md:46]:
```suggestion
Once you have completed the mapping you can send a test event. After you submit a test event, you can verify everything is set up correctly by looking for the contact on the [Audience](https://app.loops.so/audience){:target="_blank"} page in your Loops account.
```

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+We have a full tutorial for creating and updating contacts [in our docs](https://loops.so/docs/add-users/segment#create-or-update-contact){:target="_blank"}.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-loops/index.md:48]:
```suggestion
Loops has a full tutorial for creating and updating contacts [in the Loops docs](https://loops.so/docs/add-users/segment#create-or-update-contact){:target="_blank"}.
```

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+
+## Sending events
+
+In Loops you can send emails [triggered by events](https://loops.so/docs/loop-builder/triggering-emails){:target="_blank"}. You can trigger these events from Segment by using the [Track method](https://segment.com/docs/connections/spec/track/){:target="_blank"}, like this:
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-loops/index.md:52]:
```suggestion
In Loops you can send emails [triggered by events](https://loops.so/docs/loop-builder/triggering-emails){:target="_blank"}. You can trigger these events from Segment by using the [Track method](/docs/connections/spec/track/), like this:
```

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+
+```javascript
+analytics.track("User Registered");
+```
+
+When you make a `track()` call, Segment will pass this event data on to Loops, which can then send emails based on [email-sending triggers](https://loops.so/docs/loop-builder/loop-triggers){:target="_blank"} you've set up in your account.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-loops/index.md:58]:
```suggestion
When you make a Track call, Segment will pass this event data on to Loops, which can then send emails based on [email-sending triggers](https://loops.so/docs/loop-builder/loop-triggers){:target="_blank"} you've set up in your account.
```

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+
+To set up event sending with Segment, go to the Mappings tab inside your Loops destination and click "New Mapping". Select "Send Event".
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-loops/index.md:60]:
```suggestion
To set up event sending with Segment, go to the Mappings tab inside your Loops destination and click **New Mapping**. Select **Send Event**.
```

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+
+In the next page, enter the name of the event you're tracking into the "Event Name" field (e.g. "User Registered" from the example above). If you have not already created the contact in Loops, you need include an email address in your mapping (Loops requires a contact for each event).
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-loops/index.md:62]:
```suggestion
In the next page, enter the name of the event you're tracking into the "Event Name" field (e.g. "User Registered" from the example above). If you have not already created the contact in Loops, you need to include an email address in your mapping, as Loops requires a contact for each event.
```

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+
+Now you are able to send a test event. You can verify that the event was triggered properly in your Loops account from the [Events](https://app.loops.so/settings?page=events){:target="_blank"} page.
+
+Read our tutorial for sending events [in our docs](https://loops.so/docs/add-users/segment#send-event){:target="_blank"}.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-loops/index.md:66]:
```suggestion
Read the tutorial for sending events [in the Loops docs](https://loops.so/docs/add-users/segment#send-event){:target="_blank"}.
```

File: src/connections/destinations/catalog/actions-loops/index.md (hunk 2 above)
Comment (rchinn-segment) [reply src/connections/destinations/catalog/actions-loops/index.md:12]:
```suggestion
You can use this Segment destination to create and update your Loops contacts and trigger email sending with events.
```

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+});
+```
+
+If the email address or user ID do not exist in your contacts, a new contact will be created. If the email address or user ID already exist, the existing contact will be updated with the data sent in the `identify()` call.
+
+Go to the Mappings tab inside your Loops destination and click "New Mapping". Select "Create or update a contact".
+
+It is important that you set up the data mapping properly in step 3 of the form. This is where you can select which data is sent on to Loops and into which fields. We provide an example default mapping in the form but you should make sure that you set up the mapping to capture the correct data into the correct fields (you may have some [custom fields in Loops](https://loops.so/docs/add-users/properties){:target="_blank"} that the default mapping doesn't cover, for example).
+
+Any custom fields you are using in Loops can be passed inside "Custom Contact Attributes".
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-loops/index.md:44]:
```suggestion
You can pass any custom fields that you're using in Loops inside "Custom Contact Attributes".
```

File: src/connections/destinations/catalog/actions-loops/index.md (hunk 13 above)
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-loops/index.md:62]:
```suggestion
In the next page, enter the name of the event you're tracking into the "Event Name" field (for example, "User Registered" from the example above). If you have not already created the contact in Loops, you need include an email address in your mapping (Loops requires a contact for each event).
```

File: src/connections/destinations/catalog/actions-loops/index.md [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+hide-dossier: true
+---
+
+{% include content/plan-grid.md name="actions" %}
+
+[Loops](https://loops.so?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target:="_blank"} is a modern email platform for SaaS, a better way to send marketing and transactional email.
+
+You can use this Segment integration to create and update your Loops contacts as well as trigger email sending via events.
+
+This destination is maintained by Loops. For any issues with the destination, [contact their Support team](mailto:help@loops.so).
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-loops/index.md:14]:
```suggestion
Loops maintains this destination. For any issues with the destination, [contact their Support team](mailto:help@loops.so).
```

**Review by rchinn-segment — COMMENTED** [review]
@forstisabella left a few suggestions in addition to what you've already commented on

**Review by rchinn-segment — APPROVED** [review]
lgtm!

**Comment by forstisabella** [comment]
hey @tcgilbert - could you take a look at this one when you get a chance?

**Review by tcgilbert — APPROVED** [review]
looks good 👍

**Pull request merged**